packages/siwx/tests/configs/CloudAuthSIWX.test.ts

```diff
@@ -137,12 +137,12 @@ Issued At: 2024-12-05T16:02:32.905Z`)
         {
           body: '{"message":"Hello AppKit!","signature":"0x3c70e0a2d87f677dc0c3faf98fdf6313e99a3d9191bb79f7ecfce0c2cf46b7b33fd4c4bb83bca82fe872e35963382027d0d18018342d7dc36a675918cb73e9061c","clientId":null}',
           headers: {
-            Authorization: 'Bearer mock_nonce_token'
+            'x-nonce-jwt': 'Bearer mock_nonce_token'
           },
           method: 'POST'
         }
       )
-      expect(setItemSpy).toHaveBeenCalledWith('@appkit/siwx-token', 'mock_authenticate_token')
+      expect(setItemSpy).toHaveBeenCalledWith('@appkit/siwx-auth-token', 'mock_authenticate_token')
```

Comment:
don't we need a test for the `Authorization` case? same for other places where this was changed